packages/docusaurus-theme-classic/src/validateThemeConfig.ts

```diff
@@ -300,6 +300,18 @@ const CustomCssSchema = Joi.alternatives()
   .try(Joi.array().items(Joi.string().required()), Joi.string().required())
   .optional();
 
+const LogoSchema = Joi.object({
+  alt: Joi.string().allow(''),
+  src: Joi.string().required(),
+  srcDark: Joi.string(),
+  width: Joi.alternatives().try(Joi.string(), Joi.number()),
+  height: Joi.alternatives().try(Joi.string(), Joi.number()),
+  href: Joi.string(),
+  target: Joi.string(),
+  style: Joi.object(),
```

Comment:
We don't particularly recommend using inline styles, it's just a convenient escape hatch, similar to other places where we just forward props. Cf navbar items: we forward `.unknown()` props including style. Here's I'd rather not forward as it's not a dom element but a `<ThemedImage>` so I just want it to be more explicit.

Using a global className just to style this footer logo feels overkill. Inline styles are not evil, and it doesn't harm much in such case.

If I had to create a global class just to apply a minor tweak, this creates friction and we now have to touch more files (CSS, config, using same global className in both...) for little ROI.

Users that really want to avoid inline styles can still do that, but it shouldn't be mandatory

Reply:
Mmm, fair.